Mathlib/Geometry/Manifold/VectorBundle/Pullback.lean

```diff
@@ -40,3 +40,43 @@ instance SmoothVectorBundle.pullback : SmoothVectorBundle F (f *ᵖ E) IB' where
     show ((e.pullback f).coordChangeL 𝕜 (e'.pullback f) b) v = (e.coordChangeL 𝕜 e' (f b)) v
     rw [e.coordChangeL_apply e' hb, (e.pullback f).coordChangeL_apply' _]
     exacts [rfl, hb]
+
+omit [SmoothManifoldWithCorners IB B] [SmoothManifoldWithCorners IB' B']
```

Comment:
I'd rather have `omit ... in` in individual statements (or global reorganization), otherwise it's hard to know what the standing assumptions are exactly.

Reply:
These instances are no longer needed anywhere in the file! (possibly a consequence of #17616).